backend/api/actions/datawork_import.py

```diff
@@ -49,13 +56,13 @@ def datawork_import(*, importer):
         )
         with open(audio_folder / "metadata.csv", encoding="utf-8") as csvfile:
             audio_raw = list(csv.DictReader(csvfile))[0]
-        audio_metadatum = AudioMetadatum.objects.create(
-            num_channels=audio_raw["nchannels"],
-            sample_rate_khz=audio_raw["dataset_fs"],
-            sample_bits=audio_raw["sound_sample_size_in_bits"],
-            start=parse_datetime(audio_raw["start_date"]),
-            end=parse_datetime(audio_raw["end_date"]),
-        )
+            audio_metadatum = AudioMetadatum.objects.create(
+                num_channels=audio_raw["nchannels"],
+                sample_rate_khz=audio_raw["dataset_fs"],
+                sample_bits=audio_raw["sound_sample_size_in_bits"],
+                start=parse_datetime(audio_raw["start_date"]),
+                end=parse_datetime(audio_raw["end_date"]),
+            )
```

Comment:
Why did you move all of this inside the `open` ? The reason it was outside was so that we could open&close the file as quickly as possible :thinking: